R/single_quotes_linter.R

```diff
@@ -15,6 +15,11 @@
 #'   linters = single_quotes_linter()
 #' )
 #'
+#' lint(
+#'   text = "\"'this is fine'\"",
```

Comment:
do `writeLines()` with this one as well, it's hard to grasp the layers of escaping happening

Reply:
also, maybe surround this with some other code? it's a little bit hard to grok when the entire expression is just a string. Even just `paste0(x, '"this is fine"')` would be better

PS isn't this example reversed, shouldn't `'` be on the outside?